server/graphql/lib.rs

```diff
@@ -423,3 +424,32 @@ pub fn attach_discovery_graphql_schema(
 async fn discovery_index(schema: Data<DiscoverySchema>, req: GraphQLRequest) -> GraphQLResponse {
     schema.execute(req.into_inner()).await.into()
 }
+
+pub struct PluginExecuteGraphql(pub Data<GraphqlSchema>);
```

Comment:
We have another executer, for reports, this one is slightly different and needs to override user_id, also this one needs to be available in boajs context